ethsigner/core/src/integration-test/java/tech/pegasys/ethsigner/jsonrpcproxy/model/response/EthResponseFactory.java

```diff
@@ -37,6 +37,10 @@ public EthSignerResponse ethSigner(final JsonRpcError error) {
         NO_HEADERS, new JsonRpcErrorResponse(DEFAULT_ID, error), HttpResponseStatus.BAD_REQUEST);
   }
 
+  public EthSignerResponse ethSigner(final JsonRpcError error, final HttpResponseStatus code) {
```

Comment:
is this needed? doesn't look it gets used

Reply:
according to my AT, its used in FailedConnectionTest, TimeoutTest and ProxyIntegrationIntegrationTest......